networking/kvm.go

```diff
@@ -420,6 +424,7 @@ func kvmTransformFlannelNetwork(net *activeNet) error {
 	net.conf.Type = n.Delegate["type"].(string)
 	net.conf.IPMasq = n.Delegate["ipMasq"].(bool)
 	net.conf.MTU = n.Delegate["mtu"].(int)
+	net.conf.IsDefaultGateway = n.Delegate["isDefaultGateway"].(bool)
```

Comment:
I mean that if `Delegate["isDefaultGateway"]` is set, you should also set `net.conf.IsGW` so code starting in https://github.com/coreos/rkt/pull/2769/files/b3878bd68de6c7009f0feb61c46ce04dcff8e327#diff-c8c480671fc15dd96fce4719a423d7c3L520 would have effect.